lib/v2/shisu/news.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const url = 'https://news.shisu.edu.cn';
+
+module.exports = async (ctx) => {
+    const { data: res } = await got(String(url));
```

Comment:
`url` is already of type `String`. There's no need to recast it to the same type.
docs: https://docs.rsshub.app/joinus/advanced/script-standard#casting
https://github.com/DIYgod/RSSHub/blob/bf1025e0e3243b870f9909780251b12dd3cbb00c/lib/v2/shisu/news.js#L6
```suggestion
    const { data: res } = await got(url);
```